app/jobs/send_message.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+class SendMessage
+  def perform
+    if Meeting.tomorrow_meeting.present?
```

Comment:
only Meeting.tomorrow_meeting may also work, please check though.

Reply:
not working